src/nodes/Geometry3D/IndexedFaceSet.js

```diff
@@ -633,7 +633,7 @@ x3dom.registerNodeType(
             {
                 if (fieldName != "coord" && fieldName != "normal" &&
                     fieldName != "texCoord" && fieldName != "color" &&
-                    fieldName != "coordIndex")
+                    fieldName != "coordIndex" && !fieldName.startsWith("attrib_"))
```

Comment:
Why use startsWith instead of != and why the underscore in "attrib_" ? I do not think there is a field with such a name ?